src/Common/Admin/Help_Hub/Resource_Data/TEC_Hub_Resource_Data.php

```diff
@@ -0,0 +1,170 @@
+	 *
+	 * Each section can be filtered independently or as a complete set.
+	 *
+	 * @since TBD
+	 *
+	 * @return array The filtered resource sections array.
+	 */
+	public function create_resource_sections(): array {
+		$main  = Tribe__Main::instance();
+		$icons = $this->get_icon_urls( $main );
+
+		// Initial data structure for resource sections.
+		return [
+			'getting_started' => [
+				[
+					'icon'  => $icons['tec_icon_url'],
+					'title' => _x( 'The Events Calendar', 'The Events Calendar title', 'tribe-common' ),
+					'link'  => 'https://evnt.is/1ap9',
+				],
+				[
+					'icon'  => $icons['ea_icon_url'],
+					'title' => _x( 'Event Aggregator', 'Event Aggregator title', 'tribe-common' ),
+					'link'  => 'https://evnt.is/1apc',
+				],
+				[
+					'icon'  => $icons['fbar_icon_url'],
+					'title' => _x( 'Filter Bar', 'Filter Bar title', 'tribe-common' ),
+					'link'  => 'https://evnt.is/1apd',
+				],
+			],
+			'customizations'  => [
+				[
+					'title' => _x( 'Getting started with customization', 'Customization article', 'tribe-common' ),
+					'link'  => 'https://evnt.is/1apf',
+					'icon'  => $icons['article_icon_url'],
+				],
+				[
+					'title' => _x( 'Highlighting events', 'Highlighting events article', 'tribe-common' ),
+					'link'  => 'https://evnt.is/1apg',
+					'icon'  => $icons['article_icon_url'],
+				],
+				[
+					'title' => _x( 'Customizing template files', 'Customizing templates article', 'tribe-common' ),
+					'link'  => 'https://evnt.is/1aph',
+					'icon'  => $icons['article_icon_url'],
+				],
+				[
+					'title' => _x( 'Customizing CSS', 'Customizing CSS article', 'tribe-common' ),
+					'link'  => 'https://evnt.is/1api',
+					'icon'  => $icons['article_icon_url'],
+				],
+			],
+			'common_issues'   => [
+				[
+					'title' => _x( 'Known issues', 'Known issues article', 'tribe-common' ),
+					'link'  => 'https://evnt.is/1apj',
+					'icon'  => $icons['article_icon_url'],
+				],
+				[
+					'title' => _x( 'Release notes', 'Release notes article', 'tribe-common' ),
+					'link'  => 'https://evnt.is/1apk',
+					'icon'  => $icons['article_icon_url'],
+				],
+				[
+					'title' => _x( 'Integrations', 'Integrations article', 'tribe-common' ),
+					'link'  => 'https://evnt.is/1apl',
+					'icon'  => $icons['article_icon_url'],
+				],
+				[
+					'title' => _x( 'Shortcodes', 'Shortcodes article', 'tribe-common' ),
+					'link'  => 'https://evnt.is/1apm',
+					'icon'  => $icons['article_icon_url'],
+				],
+			],
+			'faqs'            => [
+				[
+					'question'  => _x( 'Can I have more than one calendar?', 'FAQ more than one calendar question', 'tribe-common' ),
+					'answer'    => _x( 'No, but you can use event categories or tags to display certain events.', 'FAQ more than one calendar answer', 'tribe-common' ),
+					'link_text' => _x( 'Learn More', 'Link to more than one calendar article', 'tribe-common' ),
+					'link_url'  => 'https://evnt.is/1arh',
+				],
+				[
```

Comment:
The resource section already has a filter. It's done in the `Hub` class. By using `tec_help_hub_resource_sections` you can filter the section and alter the data as needed. I figured it would be better to have the filter in the Hub class over the Resource Data class.

https://github.com/the-events-calendar/tribe-common/blob/feature/help-hub-iframe/src/Common/Admin/Help_Hub/Hub.php#L209

I will introduce another filter for more granular control over each section. `tec_help_hub_resource_sections{$data_class_name}`